src/components/List.js

```diff
@@ -1,14 +1,27 @@
 import React from 'react';
+import { useHistory } from 'react-router-dom';
 import useFirestore from '../hooks/useFirestore';
 
 const List = ({ token }) => {
   const { docs } = useFirestore(token);
+  const history = useHistory();
+
   return (
     <div>
       <h1>List</h1>
+      {docs.length === 0 && (
+        <div>
+          <p>Your shopping list is currently empty.</p>
+          <button onClick={() => history.push('/add-item')}>Add an Item</button>
```

Comment:
From an accessibility perspective, you might wish to consider using an anchor tag here instead of a button. 
Why is that? Basically, it comes down to HTML semantics, and how screen-readers treat buttons vs links.
In general, if something is a link to another page or route, or to an external site, you should use an anchor.
If something controls a form, opens or closes a dialog, or does anything else dynamic inside the page, use a button.
Check out this article for more in-depth discussion: https://a11y-101.com/design/button-vs-link

That being said, it's not a trivial amount of work to style a link to _look_ like a button. I'd say this is an optional suggestion, though so feel free to skip if you want. It does work as-is just fine.

Reply:
Thanks for sharing this information, Chris! Good to know 🙂

Reply:
Rachel and I updated the button to a link, but it sounds like the article is advising against styling a link to look like a button.